Sources/swift-test/usage.swift

```diff
@@ -35,7 +35,7 @@ enum Mode: Argument, Equatable, CustomStringConvertible {
         case "--help", "--usage", "-h":
             self = .Usage
         default:
-            return nil
+            self = .Run(argument)
```

Comment:
It's a protocol requirement: https://github.com/apple/swift-package-manager/blob/cc320afb89a6f2b9e97bb428f02c1733ff8f3427/Sources/OptionsParser/Argument.swift#L18